db/tasks.php

```diff
@@ -33,7 +33,7 @@
         'minute' => '0',
         'hour' => '*',
         'day' => '*',
-        'dayofweek' => '*',
-        'month' => '*'
+        'month' => '*',
```

Comment:
This can't hurt, but also this pattern will be wrong in so many places its not really worth worrying about given that it's cosmetic

Reply:
May as well. As you said, it can't hurt.